web/themes/contrib/civictheme/includes/mobile_navigation.inc

```diff
@@ -39,6 +39,30 @@ function _civictheme_preprocess_block__civictheme_mobile_navigation(array &$vari
     'bottom_menu' => 'field_c_b_bottom',
   ];
 
+  // Load all blocks of type 'civictheme_search'
+  $block_storage = \Drupal::entityTypeManager()->getStorage('block_content');
+  $blocks = $block_storage->loadByProperties(['type' => 'civictheme_search']);
+
+  // Initialize an empty array to store the search link.
+  $search_link = [];
+
+  // Iterate through the blocks and do something with them.
+  /** @var \Drupal\block_content\Entity\BlockContent $block */
+  foreach ($blocks as $block) {
+    // Get the field values directly from the block content entity.
+    $link_in_mobile_menu = $block->get('field_c_b_link_in_mobile_menu')->value ?? FALSE;
```

Comment:
This should use the `civictheme_get_field_value`

Reply:
We should also add cache depdencies for all search blocks so that if any change we invalidate ie in case they add this link